cgi-bin/dashboard-aii.cgi

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/perl -T
+# ${license-info}
+# ${developer-info
+# ${author-info}
+# ${build-info}
+
+use Config::Simple;
+use JSON::XS;
+use Switch;
+use CGI;
+
+# Configuration file to boot from HD. Following pxelinux convention
+# it must be called default so if the <IpAddressInHex> link
+# is missing the node will boot from the hard disk
+my $boothd = "default";
+
+# Quattor profiles
+my $web_root = "/var/www/html";
+my $profiles_path = "profiles";
```

Comment:
@stdweird I changed to use this parameter, this force me to allow only file:// protocol. If I want to handle http://, I need to use LWP to get profiles. I suggest to leave this as is and I'll figure a way to work with the AII framework itself for cleaner handling. What do you think ?